source/constraints/MethodConstraints.cpp

```diff
@@ -657,6 +685,16 @@ std::unique_ptr<MethodConstraint> MethodConstraint::from_json(
         index,
         ParameterConstraint::from_json(
             JsonValidation::object(constraint, /* field */ "inner")));
+  } else if (constraint_name == "any_parameter") {
+    std::optional<ParameterPosition> index = std::nullopt;
```

Comment:
Please use `JsonValidation::check_unexpected_members` here so we can catch invalid members and typos.